### PR TITLE
Add Fukuoka Ruby Award 2017 news (en)

### PR DESCRIPTION
**Pull request by digitalfukuoka**
Add Fukuoka Ruby Award 2017 news. Please review and merge.

### CODE REVIEW

File: en/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md [hunk 1]
```diff
@@ -0,0 +1,56 @@
...
+
+Entry Deadline: December 27, 2016
+
+Matz and a group of panelists will select the winners of the Fukuoka Competition. The grand prize for the Fukuoka Competition is 1 million yen. Past grand prize winners include Rhomobile (USA) and APEC Climate Center (Korea).
+
+[http://myfukuoka.com/category/news/ruby-news/](http://myfukuoka.com/category/news/ruby-news/)
+
+Programs entered in the competition do not have to be written entirely in Ruby but should take advantage of the unique characteristics of Ruby.
+
+Projects must have been developed or completed within the past 12 months to be eligible. Please visit the following Fukuoka website for additional details or to enter:
```
Comment (hlcfan) [inline en/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: websites? I see 2 links below.

**Comment by hlcfan** [comment]
If you can paste that poster or logo in this article, it'll be better?

File: en/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md (hunk 1 above)
Comment (digitalfukuoka) [inline en/news/_posts/2016-10-20-fukuoka-ruby-award-2017.md]: Thanks a lot!

**Comment by hlcfan** [comment]
Looking good to me 👍 
Can you please squash the commits?

**Comment by stomar** [comment]
@hlcfan Thanks for reviewing. Though there was an understanding that only editorial team members merge `en` content, and language maintainers only handle translations into their language.

(Regarding the commit message: I don't think it is necessary for merges like this to keep a commit message body that lists individual squashed commits.)

**Comment by hlcfan** [comment]
@stomar Thank you for reminding. Yes, you're right about the squashed commits message. I'll follow that understanding next time.

**Comment by digitalfukuoka** [comment]
It seems like the news is not published yet. Is there anything else I have to do?

**Comment by JuanitoFatas** [comment]
> It seems like the news is not published yet. Is there anything else I have to do?

Sorry for the inconvenience. I have deployed your changes. 

http://www.ruby-lang.org/en/news/2016/10/20/fukuoka-ruby-award-2017/

Thanks!

**Pull request merged**